src/codeu/chat/client/core/UserContext.java

```diff
@@ -97,8 +97,6 @@ public User removeConvoInterest(String title) {
 	public ConversationContext start(String name) {
 		final ConversationHeader conversation = controller.newConversation(
 				name, user.id);
-		// sets creator's id to creator in hashmap
-		conversation.setAccessOf(user.id, UserType.CREATOR);
```

Comment:
Correct, it wasn't needed since owner is passed as a parameter when creating a convo